### DISCUSSION TITLE
multiple project with possible same variable

### DISCUSSION THREAD

**Discussion by jiajaime** [vercel/vercel#5790] [Help] [answered]
I am making a simple game with code I downloaded from github, the game allows re-skin so I am going to have three version of the same game running with different skin and likely same variable name.  I got one of it to run. I am unsure how I can setup my vercel so multiple copy of the same code with different name can run at the same time without interfering with each other?

**Comment by mcsdevv** [comment, marked as answer]
Hi there, for this, you can use the [monorepo support](https://vercel.com/docs/git#monorepos).

You can import the same repository multiple times, giving different values to the same environment variable names for each import.
